### PR TITLE
Adds Redis Database Course for Devs in English

### PR DESCRIPTION
**Pull request by Sushant-1999**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Sushant-1999** [comment]
@eshellman  please merge the PR I have added Redis Database Course in English which is very useful for developers. Let me know if any fixes. Thanks!

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -1056,6 +1057,21 @@
 * [R Tutorial For Beginners \| Edureka](https://www.youtube.com/watch?v=fDRa82lxzaU) - Edureka!
 
 
+### Redis
+
+* [Learning Redis Tutorial](https://www.youtube.com/playlist?list=PLTgRMOcmRb3Mt3iBO2eosx5vXHeaM92TG) - Packt Video
+* [Redis and Redis Sentinel Crash Course - Redis Commands and High Availability tutorial](https://www.youtube.com/watch?v=-a07Ief51H4) - Code Academia
+* [Redis Beginner Tutorials](https://www.youtube.com/playlist?list=PLhW3qG5bs-L8n1fsiT8z_VnDhnUk4vaVq) - Automation Step by Step
+* [Redis CLI Tutorial | Complete Course](https://www.youtube.com/watch?v=0wqZd3zwh7Y) - Daily Code Buffer
```
Comment (eshellman) [inline courses/free-courses-en.md:1065]:
```suggestion
* [Redis CLI Tutorial \| Complete Course](https://www.youtube.com/watch?v=0wqZd3zwh7Y) - Daily Code Buffer
```

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -1056,6 +1057,21 @@
...
+* [Redis Course - In-Memory Database Tutorial](https://www.youtube.com/watch?v=XCsS_NVAa1g) - freeCodeCamp.org
+* [Redis Crash Course](https://www.youtube.com/playlist?list=PLoAsubXIl8uKqhvGFeH8g_gzHPwyFoVJQ) - CodeWithTim
+* [Redis Crash Course - the What, Why and How to use Redis as your primary database](https://www.youtube.com/watch?v=OqCK95AS-YE) - TechWorld with Nana
```
Comment (eshellman) [inline courses/free-courses-en.md:1068]: not long enough
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -1056,6 +1057,21 @@
...
+* [Redis Crash Course Tutorial](https://www.youtube.com/watch?v=Hbt56gFj998) - Traversy Media
```
Comment (eshellman) [inline courses/free-courses-en.md:1069]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -1056,6 +1057,21 @@
...
+* [Redis In-Memory Database Crash Course](https://www.youtube.com/watch?v=V7FPk4J10KI) - Hussein Nasser
```
Comment (eshellman) [inline courses/free-courses-en.md:1070]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

**Review by eshellman — CHANGES_REQUESTED** [review]

**Comment by Sushant-1999** [comment]
@eshellman  Please merge the PR I have fixed the changes I have included only playlists now! Please merge it soon!!

File: courses/free-courses-en.md [hunk 5]
```diff
@@ -1056,6 +1057,21 @@
 * [R Tutorial For Beginners \| Edureka](https://www.youtube.com/watch?v=fDRa82lxzaU) - Edureka!
 
 
+### Redis
+
+* [Learning Redis Tutorial](https://www.youtube.com/playlist?list=PLTgRMOcmRb3Mt3iBO2eosx5vXHeaM92TG) - Packt Video
+* [Redis and Redis Sentinel Crash Course - Redis Commands and High Availability tutorial](https://www.youtube.com/watch?v=-a07Ief51H4) - Code Academia
```
[the remark this answers is not in the document]
Comment (Sushant-1999) [reply courses/free-courses-en.md]: Done

File: courses/free-courses-en.md (hunk 1 above)
Comment (Sushant-1999) [reply courses/free-courses-en.md]: Done

File: courses/free-courses-en.md [hunk 6]
```diff
@@ -1056,6 +1057,21 @@
...
+* [Redis Beginner Tutorials](https://www.youtube.com/playlist?list=PLhW3qG5bs-L8n1fsiT8z_VnDhnUk4vaVq) - Automation Step by Step
+* [Redis CLI Tutorial | Complete Course](https://www.youtube.com/watch?v=0wqZd3zwh7Y) - Daily Code Buffer
+* [Redis Course - In-Memory Database Tutorial](https://www.youtube.com/watch?v=XCsS_NVAa1g) - freeCodeCamp.org
+* [Redis Crash Course](https://www.youtube.com/playlist?list=PLoAsubXIl8uKqhvGFeH8g_gzHPwyFoVJQ) - CodeWithTim
+* [Redis Crash Course - the What, Why and How to use Redis as your primary database](https://www.youtube.com/watch?v=OqCK95AS-YE) - TechWorld with Nana
+* [Redis Crash Course Tutorial](https://www.youtube.com/watch?v=Hbt56gFj998) - Traversy Media
+* [Redis In-Memory Database Crash Course](https://www.youtube.com/watch?v=V7FPk4J10KI) - Hussein Nasser
+* [Redis Tutorial for Beginners](https://www.youtube.com/playlist?list=PLS1QulWo1RIYZZxQdap7Sd0ARKFI-XVsd) - ProgrammingKnowledge
+* [What is Redis and What Does It Do?](https://www.youtube.com/watch?v=8A_iNFRP0F4) - CBT Nuggets
```
[the remark this answers is not in the document]
Comment (Sushant-1999) [reply courses/free-courses-en.md]: removed

File: courses/free-courses-en.md (hunk 2 above)
Comment (Sushant-1999) [reply courses/free-courses-en.md]: Removed

File: courses/free-courses-en.md (hunk 3 above)
Comment (Sushant-1999) [reply courses/free-courses-en.md]: Removed

File: courses/free-courses-en.md (hunk 4 above)
Comment (Sushant-1999) [reply courses/free-courses-en.md]: Removed

**Comment by Sushant-1999** [comment]
Thanks @eshellman

**Pull request merged**